### PR TITLE
[next/neo] update to neogradle 7.0.43

### PR DESCRIPTION
**Pull request by Cheos137**
and thus switching from mcp to neoform

### CODE REVIEW

File: src/main/java/org/parchmentmc/librarian/neogradle/ParchmentMappingVersion.java [hunk 1]
```diff
@@ -67,14 +67,14 @@ public String mcVersion() {
     }
 
     @Nonnull
-    public String mcpVersion() {
-        return mcpVersion;
+    public String mneoformVersion() {
```
Comment (XFactHD) [inline src/main/java/org/parchmentmc/librarian/neogradle/ParchmentMappingVersion.java]: You got a sneaky typo here, this should be `neoformVersion()` without a leading `m`.

Comment (Cheos137) [reply src/main/java/org/parchmentmc/librarian/neogradle/ParchmentMappingVersion.java]: thanks, am actually surprised i didn't catch that one earlier

**Comment by sciwhiz12** [comment]
@ParchmentMC/external-tools, should we permanently move Librarian to target NeoGradle (as the base branch of this PR does), or should we keep them co-existing, such that we have Librarian-FG and Librarian-NG?

I personally think we should go for the latter, in the interests of those moddings on versions where their preferred choice is Forge (and ForgeGradle) and of keeping the ability to update Librarian for those versions if we do more changes in the future.

**Comment by marchermans** [comment]
In practice I want this code to live in NeoGradle anyway
And not in Librarian.

**Comment by marchermans** [comment]
This has been implemented natively in NG7+

**Pull request closed without merging**